README.adoc

```diff
@@ -0,0 +1,121 @@
+//Ref : https://gist.github.com/dcode/0cfbf2699a1fe9b46ff04c41721dda74
+= OpenBangla Keyboard (OBK)
+ifdef::env-github[]
+:imagesdir:
+ https://gist.githubusercontent.com/path/to/gist/revision/dir/with/all/images
+:tip-caption: :bulb:
+:note-caption: :information_source:
+:important-caption: :heavy_exclamation_mark:
+:caution-caption: :fire:
+:warning-caption: :warning:
+endif::[]
+ifndef::env-github[]
+:imagesdir: .
+endif::[]
+:toc:
+:toc-placement!:
+
+//HTML for formating the logo
+
+++++
+<p align="center">
+<img src="https://github.com/OpenBangla/OpenBangla-Keyboard/raw/master/data/128.png">
+</p>
+++++
```

Comment:
How about using [image](http://gist.asciidoctor.org/?github-asciidoctor%2Fdocgist%2F%2Fgists%2Fimages.adoc#images) with [align attribute](https://asciidoctor.org/docs/user-manual/#positioning-attributes)?

Reply:
It does not work on github